text/0803-type-ascription.md

```diff
@@ -172,24 +172,20 @@ lvalue position), then we don't have the soundness problem, but we do get the
 unexpected result that `&(x: T)` is not in fact a reference to `x`, but a
 reference to a temporary copy of `x`.
 
-The proposed solution is that type ascription expressions are rvalues, but
-taking a reference of such an expression is forbidden. I.e., type asciption is
-forbidden in the following contexts (where `<expr>` is a type ascription
-expression):
+The proposed solution is that type ascription expressions are lvalues. If the
```

Comment:
Update: I realized that since lvalue controls what you can assign to, it definitely seems wrong to say that type ascription is always an lvalue, since that would permit `foo(): T = 5`.

I prefer just saying that it is an lvalue/rvalue based on the underlying expression, and that `x: T = 5` is a reference context (and hence this annotation is basically an assertion that the type of `x` is `T`).